api/odl.py

```diff
@@ -1409,11 +1409,17 @@ def _detail_for_elementtree_entry(cls, parser, entry_tag, feed_url=None):
                 if copies_tags:
                     copies_tag = copies_tags[0]
                     licenses_available = copies_tag.attrib.get("available")
+                    if licenses_available != None:
+                        licenses_available = int(licenses_available)
```

Comment:
I don't understand why this started failing.

Reply:
I chased this down for a bit and couldn't figure it out either.